src/ConcordiumNetSdk/Client/RawClient.cs

```diff
@@ -0,0 +1,841 @@
+using Concordium.V2;
+using Grpc.Core;
+using Grpc.Net.Client;
+
+namespace ConcordiumNetSdk.Client;
+
+/// <summary>
+/// The "raw" GRPC client interface exposing wrappers that abstract away
+/// connection handling and parameters present in generated methods.
+/// </summary>
+public class RawClient : IDisposable
```

Comment:
I guess this could technically be suited as a nested class of `ConcordiumClient`, however it was a bit hard on the eyes IMO, so I factored it out.

Reply:
Ok, but could we make it not public??

Reply:
Definitely.